jkube-kit/common-maven/src/main/java/org/eclipse/jkube/kit/common/util/MavenUtil.java

```diff
@@ -274,6 +277,27 @@ public static String getVersion(String groupId, String artifactId) throws IOExce
         return version;
     }
 
+    /**
+     * Return all properties in Maven project, merged with all System properties (-D flags sent to Maven).
+     * <p>
```

Comment:
looks like this came through autoformat

Reply:
This was already here:
https://github.com/eclipse/jkube/blob/08a546bb0dbb8bf6cf2949a8f549193b144b848e/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/EnvUtil.java#L228

I don't know if we have any javadoc guidelines, I usually prefer the way defined in Spring code style guidelines (but doesn't make any difference):
```
Return all properties...

<p> System proper
```

Most of our code is using this style, so I would let it as it is.